tests/unit/TestResultsDao.unitTest.ts

```diff
@@ -173,20 +174,28 @@ describe("Test Results DAO", () => {
     it("builds correct query", () => {
       const lambdaStub  = jest.fn().mockReturnValue(undefined);
       LambdaService.invoke = lambdaStub;
-
-      dao.getTestCodesAndClassificationFromTestTypes("testTypeIdVal", "vehicleTypeVal", "vehicleSizeVal", "vehicleConfigurationVal", 17, null, null, null, null);
+      const params: TestTypeParams = {vehicleType: "vehicleTypeVal",
+      vehicleSize: "vehicleSizeVal",
+       vehicleConfiguration: "vehicleConfigurationVal",
+       vehicleWheels: 17,
+       vehicleAxles: null,
+       vehicleClass: null,
+       vehicleSubclass: null,
+       euVehicleCategory: null
+      };
+      dao.getTestCodesAndClassificationFromTestTypes("testTypeIdVal", params);
 
       const callEvent = lambdaStub.mock.calls[0][1];
       expect(callEvent.path).toEqual("/test-types/testTypeIdVal");
       const expectsQueryParams = {
         euVehicleCategory: null,
         vehicleClass: null,
         vehicleSubclass: null,
-        vehicleWheels: null,
+        vehicleWheels: 17,
```

Comment:
why are you swapping vehicleWheels and vehicleAxles here?

Reply:
This unit test data was invalid